test/config-next/expiration-mailer.json

```diff
@@ -13,10 +13,10 @@
     "emailTemplate": "test/example-expiration-template",
     "debugAddr": ":8008",
     "saService": {
-      "serverAddresses": ["boulder:9095"],
+      "serverAddresses": ["sa.boulder:9095"],
       "serverIssuerPath": "test/grpc-creds/minica.pem",
-      "clientCertificatePath": "test/grpc-creds/boulder-client/cert.pem",
-      "clientKeyPath": "test/grpc-creds/boulder-client/key.pem",
+      "clientCertificatePath": "test/grpc-creds/sa.boulder/cert.pem",
```

Comment:
I think the `clientCertificatePath` and `clientKeyPath` for this configuration are wrong. You're using the `sa.boulder` directory but I think it should be an expiration mailer client key & certificate, no?